src/Microsoft.Health.Dicom.Core/Features/Context/IDicomRequestContext.cs

```diff
@@ -28,4 +28,9 @@ public interface IDicomRequestContext : IRequestContext
 
     // Opportunity for the core to change based on the caller version
     int Version { get; set; }
+
+    /// <summary>
+    /// Egress bytes from Dicom server to other resources
+    /// </summary>
+    long TotalEgressBytes { get; set; }
```

Comment:
Let's perhaps call this `TotalStorageEgressBytes` to disambiguate it from egressing data from the service overall